### DISCUSSION TITLE
被某些软件误报为病毒

### DISCUSSION THREAD

**Discussion by daguodong2024** [gkd-kit/gkd#794] [Q&A] [answered]
pc下载的时候，被杀毒了。微软自带杀毒软件。请解决。谢谢。

**Comment by daguodong2024** [comment]
![ggg](https://github.com/user-attachments/assets/af42e355-e51b-4dc6-8557-94f5e20ded3f)
![ggg](https://github.com/user-attachments/assets/0c8bb437-96e0-4bb1-b9fc-4c681e8d5957)

**Comment by lisonge** [comment, marked as answer]
你好，我不知道怎么回事，应该是你的软件误报了，我不知道怎么解决

**Reply by cvpv** [reply]
首先，我是一个路人，我正在同性交友平台github上看到这篇帖子，所以我来回答。
因为我是一个路人，所以我完全没有研究`gkd-kit`项目的打包机制和代码内容。但是我还是想回答，因为这是一个开源项目，所以我主观排除有恶意代码的选项。所以可能有以下原因：

1.  包名和微软数据库中某些记录的恶意软件重复，或者使用了共享的证书
2.  因为代码未加固或混淆不足，被微软识别到一些不好的代码特征，如特定字符串、文件哈希值，可能被安全软件误判
3.  微软Defender的误报机制，请你自己去https://www.microsoft.com/en-us/wdsi/filesubmission申诉
4.  我觉得最大的原因还是微软防火墙嫉妒我们家哥哥， 下次哥哥下的蛋，一个也不给微软吃！
